src/external/bot-skeleton/services/api/api-base.ts

```diff
@@ -56,6 +56,22 @@ class APIBase {
     is_authorized = false;
     active_symbols_promise: Promise<void> | null = null;
     common_store: CommonStore | undefined;
+    landing_company: string | null = null;
+
+    //TODO : Need to remove this api call because we have it in client store
+    async getLandingCompany() {
+        if (!this.api || !this.account_info?.country) {
+            return null;
+        }
+        try {
+            const landing_company = await this.api.send({ landing_company: this.account_info.country });
+            this.landing_company = landing_company;
+        } catch (error) {
+            console.error('Error fetching landing company:', error);
+            this.landing_company = null;
+        }
+        return this.landing_company;
+    }
```

Comment:
@rupato-deriv let's not create this method here. We already have the landing company data in store. Let's just use it from there.